packages/airnode-node/src/handlers/test-api.ts

```diff
@@ -17,20 +17,17 @@ export async function testApi(
 
   const logOptions = logger.buildBaseOptions(config, { requestId: testCallId });
 
-  const rrpTrigger = find(config.triggers.rrp, ['endpointId', endpointId]);
-  if (!rrpTrigger) {
+  const httpTrigger = find(config.triggers.http, ['endpointId', endpointId]);
+  if (!httpTrigger) {
     return [new Error(`No such endpoint with ID '${endpointId}'`), null];
```

Comment:
The phrasing on this error message is a bit weird to me. I would rather say something like:

```suggestion
    return [new Error(`Unable to find endpoint with ID:'${endpointId}'`), null];
```

I also tried to have messages/errors use colons when printing IDs/value, without spaces. Not in the scope of this PR but maybe we should look at consistency for errors/logs